packages/flutter/lib/src/gestures/scale.dart

```diff
@@ -730,6 +740,7 @@ class ScaleGestureRecognizer extends OneSequenceGestureRecognizer {
           focalPoint: _currentFocalPoint!,
           localFocalPoint: _localFocalPoint,
           pointerCount: pointerCount,
+          sourceTimeStamp: _initialEventTimestamp,
         ));
       });
     }
```

Comment:
nit: should probably set `_initialEventTimestamp` to `null` at the end of this method.

Reply:
woops, committed too fast. Should be fixed now.